tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -161,17 +168,8 @@ describe(__filename, () => {
       .toHaveProp('href', '/addon/chill-out/eula/');
   });
 
-  it('does not render an add-on ID if none exists', () => {
```

Comment:
I can't think of any situation where this would happen

Reply:
Oh, that's true. Dunno what I was thinking. 🙈